clicache/acceptance-test/SNITests.cs

```diff
@@ -34,6 +34,8 @@ public class SNITests : TestBase, IDisposable
 
         public SNITests(ITestOutputHelper testOutputHelper) : base(testOutputHelper)
         {
+			Dispose();
```

Comment:
does this make sense for C# language?  using the `Dispose` method...?

Reply:
Yeah, please create a utility method to call from Dispose, and call that here.  Calling Dispose directly is legal, but technically misuse of the semantics of IDisposable.

Reply:
Agreed. Though legal, it looks a bit weird to call Dispose directly.